app/src/shared/components/ModalDialog/index.jsx

```diff
@@ -17,13 +19,37 @@ export type InternalProps = Props & {
     children: Node,
 }
 
-const ModalDialog = ({ onClose, children, className, backdropClassName }: InternalProps) => (
+const Fullpage = styled.div`
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-color: rgba(239, 239, 239, 0.98);
+    z-index: 1;
+    overflow-y: scroll;
+`
+
+const ModalDialog = ({
+    onClose,
+    children,
+    className,
+    backdropClassName,
+    fullpage,
+}: InternalProps) => (
     <ReactModal2
         onClose={onClose}
         backdropClassName={backdropClassName}
         modalClassName={className}
     >
         <BodyClass className={NO_SCROLL} />
+        {fullpage ? (
+            <Fullpage>
+                {children}
+            </Fullpage>
+        ) : (
+            children
+        )}
         {children}
```

Comment:
This extra `{children}` causes duplication probably

Reply:
What a slip. 🤦 Hahaha!